src/utils/commits-utils.ts

```diff
@@ -14,7 +14,8 @@ export const statuses = [
 export const getCommitSha = (pipelineRun: PipelineRunKind) =>
   pipelineRun?.metadata.labels?.[PipelineRunLabel.COMMIT_LABEL] ||
   pipelineRun?.metadata.labels?.[PipelineRunLabel.TEST_SERVICE_COMMIT] ||
-  pipelineRun?.metadata.annotations?.[PipelineRunLabel.COMMIT_ANNOTATION];
+  pipelineRun?.metadata.annotations?.[PipelineRunLabel.COMMIT_ANNOTATION] ||
+  pipelineRun?.metadata?.annotations?.[PipelineRunLabel.TEST_SERVICE_COMMIT];
```

Comment:
this is also part of the fix, because I noticed that `pac.test.appstudio.openshift.io/sha` could be part of the `annotations` object instead of `labels`:

![pipelinerun-commit-sha-in-annotation](https://github.com/user-attachments/assets/0d83a9f1-652b-46ea-ba1c-24745a782a92)

at least it happened in the scenario I was trying to reproduce within my namespace :)